examples/05_glm_second_level/plot_second_level_one_sample_test.py

```diff
@@ -18,6 +18,8 @@
 
 """
 
+
+import itertools
```

Comment:
Was this needed ? I don't see the corresponding removal...

Reply:
I meant, I don't see where the initial import statement was before (must be tired... 😅 )

Reply:
moved it closer to where it is used